exonum-java-binding/core/src/main/java/com/exonum/binding/service/adapters/UserServiceAdapter.java

```diff
@@ -170,16 +172,46 @@ public void mountPublicApiHandler(long nodeNativeHandle) {
     try {
       checkState(node == null, "There is a node already (%s): are you calling this method twice?",
           node);
+
+      // Create the API handlers of this service
       node = new NodeProxy(nodeNativeHandle);
       Router router = server.createRouter();
       service.createPublicApiHandlers(node, router);
+
+      // Mount the service handlers
       server.mountSubRouter(serviceApiPath(), router);
+
+      // Log the endpoints
+      logApiMountEvent(router);
     } catch (Exception e) {
       logger.error("Unexpected exception occurs at mountPublicApiHandler", e);
       throw e;
     }
   }
 
+  private void logApiMountEvent(Router router) {
+    List<Route> serviceRoutes = router.getRoutes();
+    if (serviceRoutes.isEmpty()) {
+      return; // nothing to do here
+    }
+
+    String serviceName = getName();
+    int port = server.getActualPort().orElse(0);
+    String mountPoint = serviceApiPath();
+    // Currently the API is mounted on *all* interfaces, see VertxServer#start
+    logger.info("{} API is mounted at :{}{}", serviceName, port, mountPoint);
+
+    // TODO: is it worth it — it is not even necessarily *GET* (and you can't filter by that)?
+    //   It can also have things like request parameters ('/foo/:bar')
+    serviceRoutes.stream()
+        .map(Route::getPath)
+        .filter(Objects::nonNull) // null routes are possible in failure handlers, for instance
+        .findAny()
+        .ifPresent(someRoute ->
+            logger.info("    E.g.: http://127.1:{}{}", port, mountPoint + someRoute)
```

Comment:
127.1?

Reply:
this logging feature would be really great.

Reply:
> 127.1?

Yes, it can be used as loopback IP address. But changed to avoid confusion and compatibility issues with anything that does not support that.